feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -269,33 +281,43 @@ func configureATE(t *testing.T, ate *ondatra.ATEDevice) []*ondatra.Flow {
 	i3.IPv4().WithAddress(ateDst2.IPv4CIDR()).WithDefaultGateway(dutDst2.IPv4)
 	i3.IPv6().WithAddress(ateDst2.IPv6CIDR()).WithDefaultGateway(dutDst2.IPv6)
 
+	t.Logf("Pushing config to ATE and starting protocols...")
+	topo.Push(t)
+	topo.StartProtocols(t)
+	//return topo
+}
+
+func createTraffic(t *testing.T, ate *ondatra.ATEDevice) []*ondatra.Flow {
+	topo := ate.Topology().New()
+	p1 := ate.Port(t, "port1")
+	p2 := ate.Port(t, "port2")
+	i1 := topo.AddInterface(ateSrc.Name).WithPort(p1)
+	i2 := topo.AddInterface(ateDst.Name).WithPort(p2)
+	i3 := topo.AddInterface(ateDst2.Name).WithPort(p2)
 	ethHeader := ondatra.NewEthernetHeader()
 	ipv4Header := ondatra.NewIPv4Header()
-	ipipHeader := ondatra.NewIPv4Header()
+	ipipHeader := ondatra.NewIPv4Header().WithSrcAddress("222.222.222.222").WithDstAddress("203.0.113.0")
```

Comment:
The static analysis is complaining that these ip addresses are violating the ip address assignment guidelines:
https://github.com/openconfig/featureprofiles/blob/main/CONTRIBUTING.md#ip-addresses-assignment

Reply:
The Test case demands to use 222.222.222.222 as Source IP to test policy forwarding. hence this IP is used. 203.0.113.0 is default Dst IP as per contributing.md . may be we need to update contribute.md to allow IP 222.222.222.222.  

 on similiar line, for non 222.222.222.222. , IP 9.9.9.9 is used.

Reply:
Please refer readme.md which specifically asks for IP 222.222.222.222 to validate policy forwarding based on Source IP. 
https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/README.md Test cases 1 to 4.

Reply:
@azubaidhar, lets use 100.64.222.222 for 222.222.222.222. Please update the [Readme](https://github.com/openconfig/featureprofiles/tree/16fcf8a94113c886829a00eb504ec387db39e538/feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection) aswell

Reply:
I understand a lot of networking tutorials use IP addresses like 1.1.1.1, 2.2.2.2, and sometimes 111.111.111.111 and 222.222.222.222 as well. But these are public IP addresses belonging to actual networking providers. We don't want any traffic test to accidentally send packets to them. This is why we require tests to use one of the widely recognized documentation or benchmarking IP ranges.

FYI, 222.222.222.222 belongs to CHINANET hebei province network.

https://bgp.he.net/ip/222.222.222.222

I recommend subdividing 203.0.113.0 per policy (/26 or even /28), or you may use the BMWG range (198.18.0.0/15) as well.

Reply:
we will use 198.18.0.0/15 instead of 222.222.222.222. for non 222.222.222.222 , will use 198.51.100.0/24.
If all agree, README.md can be updated with these IPs instead of 222.222.222.222.

Reply:
I am fine with that proposal.

Reply:
Done in new commit.